README.md

```diff
@@ -1,75 +1,169 @@
+# Create the virtual environment (assuming you have Python 3.6+)
+python3 -m venv my-venv
+
+# Enable the environment
+source my-venv/bin/activate       # sh/bash/zsh
+source my-venv/bin/activate.csh   # csh/tcsh
+source my-venv/bin/activate.fish  # fish
+
+# Install ERT Storage into the environment
+pip install ert-storage  # From PyPI
+pip install -e .         # From current directory, if you've cloned this repository.
+                         # -e or --edit is the edit-mode, which allows you to develop without pip reinstalling
 ```
 
-Then, initialize the database and configure a user with a password.
+Then start ERT Storage simply by using the new `ert-storage` command. This command will tell you that you are using an in-memory SQLite database. Now, using your browser, navigate to http://localhost:8000 to bring up the Swagger documentation page. Pages of note are:
 
-``` sh
-# Initialise stuff in /var
-sudo su -i -l postgres -- initdb
+- http://localhost:8000/doc Swagger
+- http://localhost:8000/redoc ReDoc (alternative to Swagger)
+- http://localhost:8000/gql GraphiQL, interactive GraphQL page
 
-# Create superuser
-sudo su -i -l postgres -- createuser -s pg_ert
+# Persisting data
 
-# Give yourself a password (default is to disallow password login)
-sudo su -i -l postgres -- psql -c "alter user pg_ert with password = '12345'\;"
+Specifying the `ERT_STORAGE_DATABASE_URL` environment variable it is possible to specify a non-in-memory database. Refer to [SQLAlchemy Documentation](https://docs.sqlalchemy.org/en/14/core/engines.html#database-urls) for all possible database URLs. However, only SQLite and PostgreSQL with default SQLAlchemy drivers are officially supported at this time.
 
-# Create a database with you as its owner
-createdb -U pg_ert db_ert
+## SQLite
+To persist a SQLite database (note the number of forward slashes `/`):
 
-# Set the database URL so that ERT Storage can connect to it
-# You will need to redo this every time you close your shell
-export ERT_STORAGE_DATABASE_URL="postgresql://pg_ert:12345@localhost:5432/db_ert"
+``` sh
+export ERT_STORAGE_DATABASE_URL="sqlite:///ert.db"       # Use ert.db in current working directory
+export ERT_STORAGE_DATABASE_URL="sqlite:////tmp/ert.db"  # Use /tmp/ert.db
 ```
 
-## Setup ERT Storage
-Install this project in a virtualenv. Among other things, this will pull and
-compile `psycopg2`, which is the PostgreSQL driver (hence the requirement for
-`postgresql-devel`).
+NOTE: SQLite does not support the same features as PostgreSQL, and as such migrations (via `alembic`) are not supported and some data may be stored or fetched in an inefficient manner. You will have to delete your data whenever we update the database schema, which happens relatively often.
+
+## PostgreSQL
+We use PostgreSQL in production and we use `alembic` to maintain 
+
 
+Set `ERT_STORAGE_DATABASE_URL`
 ``` sh
-pip install -e .[test,postgres]
+# Example postgres settings
```

Comment:
Can we actually have a dummy config file in the repo, eg. `.config_db` with some default values?

Reply:
We'll need some sort of pipeline to verify that this dummy config file is up-to-date with the current database schema. I think it's beyond the scope of this PR at least.